internal/component/prometheus/exporter/cloudwatch/config.go

```diff
@@ -55,14 +56,16 @@ type DecoupledScrapeConfig struct {
 type TagsPerNamespace = cloudwatch_exporter.TagsPerNamespace
 
 // DiscoveryJob configures a discovery job for a given service.
+// TODO: Add a recently_active_only attribute.
```

Comment:
Nit: any reason why not adding this now that could be included in this comment? If not, I'm okay to leave as-is and not scope creep adding it.

Reply:
It's probably safe to add, but I want to check with the Cloud Integrations team first, in case they didn't add it on purpose.